cmd/root.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+
+package cmd
+
+import (
+	"sort"
+
+	"github.com/elastic/elastic-agent-changelog/internal/cobraext"
+
+	"github.com/spf13/cobra"
+)
+
+var commands = []*cobraext.Command{}
+
+// RootCmd creates and returns root cmd for elastic-agent-changelog.
+func RootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:          "elastic-agent-changelog",
+		Short:        "elastic-agent-changelog - Command line tool used for managing the change log for Elastic Agent and related components, including Beats.",
+		SilenceUsage: true,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return cobraext.ComposeCommandActions(cmd, args)
+		},
```

Comment:
This is a `noop` so please either remove it or use it.

Reply:
Correct, will remove it.